src/components/Clothe.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components/macro'
+
+function Clothe({ imageUrl }) {
+  return <StyledImage src={imageUrl} alt="" />
+}
+
+const StyledImage = styled.img`
+  width: 200px;
+`
+
+export default Clothe
```

Comment:
```suggestion
export default Clothing
```

Remember to also change the name of the file!